src/main/java/frc/robot/Robot.java

```diff
@@ -15,8 +18,13 @@
  * project.
  */
 public class Robot extends TimedRobot {
-  private Command m_autonomousCommand;
-
+  private static final String kDefaultAuto = "RamAuto";
+  private static final String kCustomAuto1 = "DoNothing";
+  private static final String kCustomAuto2 = "InNOut";
+  private static final String kCustomAuto3 = "GrabBunny";
+  private static final String kCustomAuto4 = "SprayNPraySeq";
+  private static final String kCustomAuto5 = "SprayNPrayFull";
```

Comment:
These constants aren't actually used in the switch case

Reply:
ok